src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModal.tsx

```diff
@@ -110,7 +110,11 @@ export const AddArtworksModal: FC<AddArtworksModalProps> = ({
             })}
           </Text>
 
-          <Button onClick={handleSave} loading={isSaving}>
+          <Button
+            onClick={handleSave}
+            loading={isSaving}
+            data-testid="artwork-list-modal-save"
```

Comment:
**suggestion**: `save-artworks-to-artwork-list-button` (feel free to ignore)

Reply:
I was optimizing for readability [over here](https://github.com/artsy/integrity/pull/488/files#diff-ee5d9c0bc9524aa43db1d6b5aa7f1ba4b6fa0c5e29443976413284b14ffc7f0fR47-R48), so I think I'll leave this.